src/jit/lsrabuild.cpp

```diff
@@ -1718,7 +1718,7 @@ void LinearScan::buildPhysRegRecords()
 //
 BasicBlock* getNonEmptyBlock(BasicBlock* block)
 {
-    while (block != nullptr && block->bbTreeList == nullptr)
+    while (block != nullptr && block->bbTreeList == nullptr && block->bbStmtList == nullptr)
```

Comment:
This is in LIR so `bbStmtList` shouldn't be relevant anymore?

Reply:
Fixed